packages/mobile/src/components/premium-track-purchase-drawer/AudioMatchSection.tsx

```diff
@@ -0,0 +1,42 @@
+import { View } from 'react-native'
+
+import { makeStyles } from 'app/styles'
+
+import { Text } from '../core/Text'
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  root: {
+    backgroundColor: palette.secondary,
+    border: palette.secondaryDark2,
+    paddingHorizontal: spacing(8),
+    paddingVertical: spacing(2)
+  },
+  text: {
+    textAlign: 'center'
```

Comment:
I just went down a rabbit hole trying to decide if this should use flexbox and `justify-content: center` instead. TL/DR: For block-level and containers with multiple items that may or may not be text, use flex. For inline content that is only text (which in this case will be a `<p>` tag rendered by the `<Text>` component), text-align is probably more important.

Reply:
edit: Also this is mobile, so the rules are probably a little different anyway 😅

Reply:
Probs the "harmony" plan for this would be have alignment be a prop to <Text so it's not left up to me to make a call 🙃  cc @Kyle-Shanks